streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -47,7 +50,7 @@
  *     final KafkaStreams streams = ...;
  *     streams.start()
  *     ...
- *     final String queryableStoreName = table.getStoreName(); // returns null if KTable is not queryable
+ *     final String queryableStoreName = table.queryableStoreName(); // returns null if KTable is not queryable
```

Comment:
Hmm, right now even if the user does not provide a queryable name in the operator that generates this KTable, we would still return the internal name in this call. Is that right?